lib/winston-logstash.js

```diff
@@ -50,6 +50,7 @@ var Logstash = exports.Logstash = function (options) {
   this.strip_colors        = options.strip_colors || false;
   this.label               = options.label || this.node_name;
   this.meta_defaults       = options.meta || {};
+  this.transformer         = options.transformer || defaultTransformer;
```

Comment:
I know this is super nitpicky, but could you change the name of this to "transform"? Seems more in line with naming conventions I've seen.

Reply:
done